crates/libtiny_tui/src/tui.rs

```diff
@@ -299,7 +299,21 @@ impl TUI {
                 // Fall through to print help for cmd.rs commands
                 CmdResult::Continue
             }
-            Some("quit") => CmdResult::Quit,
+            Some("quit") => {
+                // Cut the "quit " part of the command
+                // (the slash is already cut outside this function),
+                // and use the rest as the message for the /quit,
+                // including other (potentially consecutive) spaces.
+                // `&cmd[("quit ".len())..]` would freeze the program
+                // if the command given has nothing more besides "/quit".
+                let reason: String = cmd.chars().skip(5 /* "quit ".len() */).collect();
```

Comment:
Just use `"quit ".len()`, the compiler should be able to optimize it to 5.

I feel like the comment here is not necessary, the code is not too complicated.

Maybe we should note here that when [`SplitWhitespace::as_str`](https://doc.rust-lang.org/core/str/struct.SplitWhitespace.html#method.as_str) is stabilized we can use it to get the rest of the string (the quit message).